Update README and CHANGELOG (#17)
answear/luigis-box-bundle: README.md

`ApiResponse::$success` will be set to `false` if any of passed documents fails. Check `$okCount` if you want to know how many documents were updated and `$errors` to check exactly which documents failed.

### Searching (documentation [here](https://live.luigisbox.com/#search-as-a-service))

1. Request

```php
use Answear\LuigisBoxBundle\ValueObject\SearchUrlBuilder;

// ...

$page = 3;
$urlBuilder = new SearchUrlBuilder('tracker-id', $page);
$urlBuilder
    ->setQuery('nice top')
    ->addFilter('type', 'product')
    ->addFilter('category', 'top')
    ->addFilter('brand', 'Medicine')
    ->addFilter('brand', 'Answear')
    ->addPrefer('brand', 'Answear')
    ->setSort('size', 'asc');

> //above produce a url like `size=10&tracker_id=tracker-id&page=3&q=nice+top&f%5B0%5D=type%3Aproduct&f%5B1%5D=category%3Atop&f%5B2%5D=brand%3AMedicine&f%5B3%5D=brand%3AAnswear&sort=size%3Aasc&prefer%5B0%5D=brand%3AAnswear`

Reviewer: ```suggestion
// code aboves produces a url like `size=10&tracker_id=tracker-id&page=3&q=nice+top&f%5B0%5D=type%3Aproduct&f%5B1%5D=category%3Atop&f%5B2%5D=brand%3AMedicine&f%5B3%5D=brand%3AAnswear&sort=size%3Aasc&prefer%5B0%5D=brand%3AAnswear`
```

/** @var \Answear\LuigisBoxBundle\Service\SearchRequest $request **/
$searchResponse = $request->search($urlBuilder);

```

> Check the Luigi's Box documentation to find out exactly what the field on `SearchUrlBuilder` is doing.

Reviewer: ```suggestion
Check the Luigi's Box documentation to find out exact purpose of each field `SearchUrlBuilder` is exposing.
```

2. Response

> `SearchRequest::search()` will return the `SearchResponse` object with fields:

Reviewer: ```suggestion
`SearchRequest::search()` will return a `SearchResponse` object with following fields:
```
